pyproject.toml

```
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "statick-planning"
+authors = [{name = "NIWC Pacific"}]
+description="Statick analysis plugins for planning files."
+version = "0.2.2"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "CC0-1.0"}
+classifiers = [
+    "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Testing",
+]
+dependencies = [
+    "importlib_metadata",
+    "statick@git+https://github.com/tdenewiler/statick@stdlib-plugins",
```

```suggestion
    "statick>=1.0.0"
```
once that PR is merged and we have a new release.